dmoj/judge.py

```diff
@@ -183,7 +183,8 @@ def begin_grading(self, id, *args, **kwargs):
         self.current_submission_thread.daemon = True
         self.current_submission_thread.start()
 
-        if is_blocking:
+        # Submission may have been killed already, but block if it hasn't been.
```

Comment:
What does this have to do with ANSI escape codes?

Reply:
It's a race. Not HTMLifying makes it trigger all the time and crashes the judge, so it is fixed here.